nowcasting_dataset/data_sources/satellite/satellite_data_source.py

```diff
@@ -145,5 +145,7 @@ def open_sat_data(zarr_path: str, consolidated: bool) -> xr.DataArray:
     # To make it easier to align the satellite data with other data sources
     # (which are at 00, 05, ..., 55 minutes past the hour) we add 1 minute to
     # the time dimension.
+    # TODO Remove this as new Zarr already has the time fixed
+    #  See https://github.com/openclimatefix/nowcasting_dataset/issues/313
     data_array["time"] = data_array.time + pd.Timedelta("1 minute")
```

Comment:
Good catch!

So the code is backwards-compatible with the "old" satellite zarrs, I wonder if we should change line 150 to something like `data_array["time"] = pd.DatetimeIndex(data_array.time).round('5 min')` which should work for both the old and new zarr?

Reply:
Yeah, sounds good!